### ISSUE TITLE
Render errors and warnings

### ISSUE THREAD

**Issue by razor-x** [seamapi/docs#333]
We will have separate errors and warnings sections that are children of the resource and look similar to the props section.

**Comment by DebbieAtSeam** [comment]
How about something like this?

```
# `acs_system`

Represents an [access control system](https://docs.seam.co/latest/capability-guides/access-systems).

## Properties

### `acs_system_id`

Format: `ID`

ID of the `acs_system`.

---

# Endpoints

## [`/acs/systems/get`](./get.md)

Returns a specified [access control system](https://docs.seam.co/latest/capability-guides/access-systems).

---

# Events

## `Event 1`

Event 1 description.

<details>

<summary>Event 1 Properties</summary>

### `event1prop1`

Format: `format`

Event 1 prop 1 description.

</details>

---

# Errors

## `Error 1`

Error 1 description.

---

# Warnings

## `Warning 1`

Warning 1 description.

---
```

Question: How do we show the format of the errors and warnings? I'm assuming that they're all based on the same error and warning objects, so all errors should have the same set of props, and all warnings should have the same set of props. Is that correct? Is there a way in which we could link to a definition of the error object and the warning object instead of repeating the same prop descriptions over and over again?

**Comment by razor-x** [comment]
> all errors should have the same set of props, and all warnings should have the same set of props. Is that correct? 

Mostly, however now that we can document and type these, I'd like errors and warnings to have more helpful props. Some already do, e.g., we have `devices_connected_to_other_account` with a `device_ids` prop.

**Comment by razor-x** [comment]
I'd say Events are linked to the route, since we have `lock.unlocked` for example. We could document `lock.unlocked` under `device` but it would make more send to consider that event under `/locks`.

**Comment by razor-x** [comment]
@andrii-balitskyi Yes, but they should be properly typed for a acs_system and acs_user. All errors and warnings will be getting proper types. Let's just render the ones that have the types, and have the docsgen just render something generic for the ones that are not.
